Framework/Kernel/inc/MantidKernel/VMD.h

```diff
@@ -234,12 +234,12 @@ template <typename TYPE = double> class DLLExport VMDBase {
     boost::split(strs, str, boost::is_any_of(", "));
```

Comment:
I don't know how large the strings usually are here, but maybe this could profit from your tokenizer?